src/ui/AriaLive/AriaLive.ts

```diff
@@ -38,6 +41,10 @@ export class AriaLive implements IAriaLive {
     root.on(QueryEvents.duringQuery, () => this.onDuringQuery());
     root.on(QueryEvents.querySuccess, (e, args: IQuerySuccessEventArgs) => this.onQuerySuccess(args));
     root.on(QueryEvents.queryError, (e, args: IQueryErrorEventArgs) => this.onQueryError(args));
+    root.on(OmniboxEvents.querySuggestSuccess, (e, args: IQuerySuggestSuccessArgs) =>
+      args.completions.length ? (this.querySuggestions = args.completions.length) : this.onNoQuerySuggest()
```

Comment:
When there are no query suggestions, the `querySuggestRendered` event isn't fired.

Unfortunately, updating the text here means that the user will be notified that there are no suggestions even if they focus out of the search box before the querySuggest query is finished. I don't think that's a very big compromise though.